client/jetpack-cloud/sections/partner-portal/license-state-filter/style.scss

```diff
@@ -0,0 +1,3 @@
+.section-nav__mobile-header-text .count {
+	margin-left: 8px;
+}
```

Comment:
This is too generic in the stylesheet for a specific component. We should either scope down the selector to our custom component (e.g. `.license-state-filter .section-nav__...` or add this fix to the SectionNav component styles (not sure how good of an idea the latter is).